### PR TITLE
Add osu!mania & osu!taiko  2022 PP/SR survey results newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**

### CODE REVIEW

File: news/2022-02-07-taiko-mania-survey-results.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+layout: post
+title: "Results - osu!taiko & osu!mania Performance Points & Star Rating Surveys"
+date: 2022-02-07 10:00:00 +0000
+---
+
+The key smashing and drum beating members of the community have had their say, and the results are in! See how your opinions on Performance Points & Star Ratings compare!
+
+![](/wiki/shared/news/2021-07-27-performance-points-star-rating-updates/star-rating.jpg)
+
+These latest surveys are born on the back of efforts by the intrepid performance developers to find out exactly where the community stands on things in the hopes of catering upcoming changes to the largest number of people possible.
```
Comment (Walavouchey) [inline news/2022-02-07-taiko-mania-survey-results.md:11]:
```suggestion
These latest surveys are born on the back of efforts by the intrepid performance point developers to find out exactly where the community stands on things in the hopes of catering upcoming changes to the largest number of people possible.
```

**Comment by Walavouchey** [comment]
questions in the results seem to be truncated

**Comment by Ephemeralis** [comment]
> questions in the results seem to be truncated

known issue and won't be addressed apparently (if it's related to things like https://puu.sh/IHA5N/b9c9d34a05.png)

File: news/2022-02-07-taiko-mania-survey-results.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+These latest surveys are born on the back of efforts by the intrepid performance point developers to find out exactly where the community stands on things in the hopes of catering upcoming changes to the largest number of people possible.
+
+If you're interested in how it all went down, check out the links below:
+
+- [Click here for the **osu!taiko** survey results!](https://assets.ppy.sh/surveys/2022-pp-taiko/viewer.htm?v=5)
+- [Click here for the **osu!mania** survey results!](https://assets.ppy.sh/surveys/2022-pp-mania/viewer.htm?v=2)
+
+Just like with the results from the [osu! survey](https://osu.ppy.sh/home/news/2021-09-02-pp-sr-survey-results), you can use the buttons on the left side to navigate through each rank group to get a better understanding of where things sit across players of varying skill and dedication levels.
+
+A big thank you to everyone who took the time out of their day to take the survey, [mangomizer](https://osu.ppy.sh/users/1893718) and [-Lawtron-](https://osu.ppy.sh/users/11475208) for designing the osu!taiko survey, and [taggerung](https://osu.ppy.sh/users/14751043) and [Quenlla](https://osu.ppy.sh/users/4725379) for designing the osu!mania survey.
```
Comment (peppy) [inline news/2022-02-07-taiko-mania-survey-results.md:20]:
```suggestion
A big thank you to peppy for taming google sheets once again.
```

**Review by peppy — APPROVED** [review]

File: news/2022-02-07-taiko-mania-survey-results.md (hunk 2 above)
Comment (peppy) [reply news/2022-02-07-taiko-mania-survey-results.md:20]: (do not apply this)

**Comment by peppy** [comment]
> > questions in the results seem to be truncated
> 
> known issue and won't be addressed apparently (if it's related to things like https://puu.sh/IHA5N/b9c9d34a05.png)

it's still legible so i don't see a need to fix it. still better than the truncation of many of the questions and answers that remains.

**Pull request merged**